org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/alloy4/WorkerEngine.java

```diff
@@ -47,8 +49,9 @@
  * subsequent task; however, if the subprocess crashed, the crash will be
  * reported to the parent process via callback, and if we try to execute another
  * task, then a new subprocess will be spawned automatically.
+ *
+ * @modified Nuno Macedo // [HASLab] electrum-unbounded
```

Comment:
the SMV backend of Pardinus launches a new process for SMV model checkers; method `destroy()` was not stopping these children process, so I had to resort to sigterm; happy to change this to a more appropriate method